src/variables/create.jl

```diff
@@ -88,37 +88,63 @@ function _create_variables_tables(connection)
         connection,
         "CREATE OR REPLACE TEMP SEQUENCE id START 1;
         CREATE OR REPLACE TABLE var_storage_level_rep_period AS
+        WITH filtered_data AS (
+            SELECT
+                t_low.asset,
+                t_low.year,
+                t_low.rep_period,
+                t_low.time_block_start,
+                t_low.time_block_end
+            FROM t_lowest_all AS t_low
+            LEFT JOIN asset
+                ON t_low.asset = asset.asset
+            WHERE
+                asset.type = 'storage'
+                AND asset.is_seasonal = false
+            ORDER BY
+                t_low.asset,
+                t_low.year,
+                t_low.rep_period,
+                t_low.time_block_start
+        )
         SELECT
-            nextval('id') as index,
-            t_low.asset,
-            t_low.year,
-            t_low.rep_period,
-            t_low.time_block_start,
-            t_low.time_block_end
-        FROM t_lowest_all AS t_low
-        LEFT JOIN asset
-            ON t_low.asset = asset.asset
-        WHERE
-            asset.type = 'storage'
-            AND asset.is_seasonal = false;
+            nextval('id') AS index,
+            asset,
+            year,
+            rep_period,
+            time_block_start,
+            time_block_end
+        FROM filtered_data
```

Comment:
There seems to be no need for the `WITH`. Adding the `ORDER BY` to the original query would suffice I think.